src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,20 @@
 import React from 'react';
 import './MoviesList.scss';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[]
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    <ul>
+      {movies.map(movie => (
+        <li key={movie.imdbId}>
+          <MovieCard movie={movie} key={movie.imdbId} />
+        </li>
```

Comment:
no need to wrap the component in a `li`, you can set `key `to the component and render `li` inside it